pkg/v1/providers/infrastructure-aws/v1.2.0/cconly/base.yaml

```diff
@@ -93,6 +93,20 @@ spec:
         default: false
   - name: network
     required: true
+    schema:
+      openAPIV3Schema:
+        type: string
```

Comment:
This is not right. 
```
- name: network
  required: true
  ```
  should be where the new `- name: proxy` is (at line 109) and the schema at line 96-98 should be removed